src/Core/Form/Entity/ResponseHandler/SuccessResponseHandlerInterface.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Form\Entity\ResponseHandler;
+
```

Comment:
as these interfaces are strongly linked to the controller, and the `HttpFoundation\Response|Request` I don't think this is the right namespace, they should be in `PrestahopBundle` with the controller

Reply:
i guess, but maybe we can keep it like this at least for now, so all files that are related to "entity forms" component are in the same namespace?

Reply:
to me it is more linked to EntityController than EntityForm, the form could be used in a command, a service or anywhere and does not need to be coupled with the response It is not its responsibility
Besides you already put the common EntityController in PrestashBundle, but we can still have a Form namespace inside the PrestashopBundle

Reply:
ResponseHandler aren't entity, they must be in Http Context instead. You can have a SuccessResponseHandler without Entity saved